src/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocketOptions.cs

```diff
@@ -15,11 +15,15 @@ public sealed class ClientWebSocketOptions
         private bool _isReadOnly; // After ConnectAsync is called the options cannot be modified.
         private readonly List<string> _requestedSubProtocols;
         private readonly WebHeaderCollection _requestHeaders;
-        private TimeSpan _keepAliveInterval;
+        private TimeSpan _keepAliveInterval = TimeSpan.FromSeconds(30); // Same as WebSocket.DefaultKeepAliveInterval.  Switch to that when it's available.
```

Comment:
Please add a `TODO` and a tracking bug otherwise we'll probably miss this comment in the future.